ninja/management/commands/export_openapi_schema.py

```diff
@@ -55,25 +64,48 @@ def add_arguments(self, parser: CommandParser) -> None:
             help="Output schema to a file (outputs to stdout if omitted).",
         )
         parser.add_argument(
-            "--indent", dest="indent", default=None, type=int, help="JSON indent"
+            "--indent",
+            dest="indent",
+            default=None,
+            type=int,
+            help="Schema indentation level",
         )
         parser.add_argument(
             "--sorted",
             dest="sort_keys",
             default=False,
             action="store_true",
-            help="Sort Json keys",
+            help="Sort schema keys",
         )
 
     def handle(self, *args: Any, **options: Any) -> None:
         api = self._get_api_instance(options["api"])
         schema = api.get_openapi_schema()
-        result = json.dumps(
-            schema,
-            cls=NinjaJSONEncoder,
-            indent=options["indent"],
-            sort_keys=options["sort_keys"],
-        )
+
+        fmt = options["format"]
+        if fmt == "json":
+            result = json.dumps(
+                schema,
+                cls=NinjaJSONEncoder,
+                indent=options["indent"],
+                sort_keys=options["sort_keys"],
+            )
+        elif fmt == "yaml":
+            try:
+                import yaml
+
+                from ninja.yaml import NinjaSafeDumper
+            except ImportError:
+                raise CommandError("Install PyYAML or django-ninja[yaml].")
+            result = yaml.dump(
+                schema,
+                Dumper=NinjaSafeDumper,
+                indent=options["indent"],
+                sort_keys=options["sort_keys"],
+            )
+        else:
```

Comment:
Do we need this else statement if argparse already raises an error?🤔

Reply:
Yeah, I wasn't sure what to do with this part. I can remove the else and keep the comment. 👍